Arcs/Arcs.cpp

```diff
@@ -60,12 +74,18 @@ PROGRAM " " PACKAGE_VERSION "\n"
 "   -m, --index_multiplicity=RANGE  barcode multiplicity range [50-10000]\n"
 "   -d, --max_degree=N    max node degree in scaffold graph [0]\n"
 "   -e, --end_length=N    contig head/tail length for masking alignments [30000]\n"
-"   -r, --error_percent=N p-value for head/tail assignment and link orientation\n"
-"                         (lower is more stringent) [0.05]\n"
+"   -r, --error_percent=N p-value for head/tail assignment and link orientation (lower is more stringent) [0.05]\n"
 "   -v, --run_verbose     verbose logging\n"
+"ARCS Specific Options:\n"
+"   -f, --file=FILE       FASTA file of contig sequences to scaffold [optional]\n"
+"   -s, --seq_id=N        (only in default mode)min sequence identity for read alignments [98]\n"
```

Comment:
What do you mean by 'only in default mode'?

Reply:
I meant ARCS mode. As we already tell it in "Options specific to ARCS" section, I see it is redundant.